src/fileoverview_comment_transformer.ts

```diff
@@ -0,0 +1,80 @@
+import * as ts from 'typescript';
+import * as jsdoc from './jsdoc';
+
+const FILEOVERVIEW_COMMENTS: ReadonlySet<string> =
+    new Set(['fileoverview', 'externs', 'modName', 'mods', 'pintomodule']);
+
+function skipLeadingCommentsRecursively(n: ts.SourceFile) {
```

Comment:
Docs?  Seems like it sets as flag on a bunch of nodes?

Reply:
actually this is unused by now. Whoops.